deploy.sh

```diff
@@ -7,13 +7,13 @@ set -eu
 
 # These parameters must be changed to your own fork 
 KAYOBE_CONFIG_REPO="https://github.com/example/stackhpc-kayobe-config"
-KAYOBE_CONFIG_BRANCH=stackhpc/yoga
```

Comment:
Possibly should have kept this change to the antelope branch but oh well. I don't think we'll come back to Yoga

Reply:
hmm, I don't think any of this is valid for yoga except the change to `stackhpc_release_pulp_url`. I'll revert.